### PR TITLE
Migrate buttons and LCD tutorial pages from the wiki

### PR DESCRIPTION
**Pull request by dwalton76**

### CODE REVIEW

File: docs/tutorials/LCD.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+provides [standard Linux framebuffer interface](https://www.kernel.org/doc/Documentation/fb/api.txt)
+and it's possible to write pixel data into `/dev/fb0` directly using `write`
+or `mmap`.
+
+## The pixel buffer format
+Each byte in the buffer represents 8 pixels. Leftmost pixel is in the least
+significant bit. Value 0 is white and value 1 is black (`fb_fix_screeninfo#visual`
+is set to `FB_VISUAL_MONO01`).
+
+Each row need to be 4 bytes aligned. So it's (178+31)/324 = 24 bytes per row
```
Comment (dlech) [inline docs/tutorials/LCD.md]: The math does not add up here. I believe it should be `(178+31)/32*4

File: docs/tutorials/buttons.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+---
+title: Using the Buttons
+subject: Buttons
+---
+
+* Table of Contents
+{:toc}
+
+![EV3 Buttons](/images/lego_ev3_buttons.jpg)
```
Comment (dlech) [inline docs/tutorials/buttons.md]: Awesome! I've been wanting to have something like this for a long time.

File: docs/tutorials/buttons.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+integer) and the last 4 bytes are the value (unsigned 32-bit integer).
+
+Here is an example. It prints out 2 lines each time you press a button on the
+EV3 and 2 more lines each time you release a button. And of course, press CTRL+C
+to end.
+
+    root@ev3dev:/sys/class/leds# hexdump -e \
+    '"timestamp:%d.%6d""\t""" 1/2 "type:%i""\t"""  1/2 "code:%3i""\t"""  "value:%d\n"' \
+    < /dev/input/by-path/platform-gpio-keys.0-event 
+    timestamp:1391366282.119886 type:1  code:  1    value:0
```
Comment (dlech) [inline docs/tutorials/buttons.md]: I think this should be `code:  14` now instead of `code:   1`.

File: docs/tutorials/buttons.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    timestamp:1391366282.120100 type:0  code:  0    value:0
+    timestamp:1391366282.829942 type:1  code:  1    value:1
```
Comment (dlech) [inline docs/tutorials/buttons.md]: ditto.

**Comment by dlech** [comment]
Thanks for doing this!

There are just a few minor issues, if you don't mind fixing them.

Also, the LCD bit could be improved using ioctls to get the screen attributes instead of hard coding values. If you are interested in learning more, I'll dig up the info, otherwise, this is probably good enough for now.

File: docs/tutorials/LCD.md (hunk 1 above)
Comment (dwalton76) [inline docs/tutorials/LCD.md]: This was all cut-n-paste but yeah the math here is hosed.  How about:

Each row needs to be aligned on a 4-byte boundry.  The closest 4-byte boundry to 178 is 192. 192/8 = 24 bytes per row

File: docs/tutorials/buttons.md (hunk 2 above)
Comment (dwalton76) [inline docs/tutorials/buttons.md]: I confess I found this via google images :)

File: docs/tutorials/buttons.md (hunk 3 above)
Comment (dwalton76) [inline docs/tutorials/buttons.md]: Good catch will fix these

File: docs/tutorials/LCD.md (hunk 1 above)
Comment (dlech) [inline docs/tutorials/LCD.md]: that works.

File: docs/tutorials/buttons.md (hunk 2 above)
Comment (dlech) [inline docs/tutorials/buttons.md]: Then I must ask, do we have permission to use this image?

**Comment by dlech** [comment]
I can't merge anymore because there were other changes made after you forked the repo. Can you rebase or merge the changes please.

File: docs/tutorials/buttons.md (hunk 2 above)
Comment (dwalton76) [inline docs/tutorials/buttons.md]: I didn't ask, it is from a robotics school in japan:
http://www.afrel.co.jp/en/archives/850

I can chop it if you want

Comment (rhempel) [inline docs/tutorials/buttons.md]: I can give you a high res image for the EV3 I use for my video tutorials - it's an LDraw rendering - just ask

Comment (dlech) [inline docs/tutorials/buttons.md]: I don't want to use anything without asking first. As an alternative, @G33kDude, has an svg of the EV3 that he made. We could probably make a similar image that looks even better than the current one.

Comment (dlech) [inline docs/tutorials/buttons.md]: @rhempel, I've been putting them at https://github.com/ev3dev/ev3dev/tree/master/logo

**Comment by dwalton76** [comment]
Ok I should be up to date now

File: docs/tutorials/buttons.md (hunk 2 above)
Comment (dwalton76) [inline docs/tutorials/buttons.md]: I understand, I've removed the image.

**Comment by dlech** [comment]
I had to rebase to remove the image from git history, but this is merged now. Thanks!

**Comment by rhempel** [comment]
I put the image of the front of the EV3 into the ev3dev meta repo

**Comment by dlech** [comment]
Thanks for that. I used it here: http://localhost:4000/docs/tutorials/using-ev3-buttons/

**Comment by rhempel** [comment]
If I ever get the time, I'd redo this using svg :-)

**Comment by G33kDude** [comment]
ev3

Not perfect, but I think it looks pretty good.

**Comment by rhempel** [comment]
Inkscape can do a pretty good job of making paths from  bitmaps if you use a good high res image. If you can use gimp, then this set of layers is pretty helpful:

[https://github.com/ev3dev/ev3dev/blob/master/images/EV3.xcf]

**Pull request closed without merging**